### ISSUE TITLE
FSE: Hide links to the customizer for those using FSE and a compatible theme

### ISSUE THREAD

**Issue by apeatling** [Automattic/wp-calypso#34791]
We do not want anyone that is using a full site editing theme to end up in the customizer through our guidance.

There are many places throughout our UI and in emails where the user is guided to the customizer. On a case by case basis these should either be removed, or replaced with a link directly into Gutenberg with the home page selected to edit.

If a user is not using a full site editing compatible theme, these customizer links should remain until we have phased out all incompatible themes.

----

**Places in our UI that link to the Customizer:**

In the sidebar (this should be removed):

<img width="348" alt="Screen Shot 2019-07-19 at 1 28 47 PM" src="https://user-images.githubusercontent.com/1464705/61564294-007cb800-aa2b-11e9-969d-b9099b7c0013.png">

---

In /themes when hovering over your active theme (this should link to Gutenberg to edit your homepage):

<img width="380" alt="Screen Shot 2019-07-19 at 1 28 56 PM" src="https://user-images.githubusercontent.com/1464705/61564325-138f8800-aa2b-11e9-8049-47cc3602c14b.png">

---

When you visit a theme page in the theme showcase (this should link to Gutenberg to edit your homepage):

<img width="662" alt="Screen Shot 2019-07-19 at 2 02 59 PM" src="https://user-images.githubusercontent.com/1464705/61565473-00ca8280-aa2e-11e9-9b3f-2cc1075627b9.png">

---

When you've activated a new theme via the theme showcase (this should link to Gutenberg to edit your homepage):

<img width="999" alt="Screen Shot 2019-07-19 at 1 29 07 PM" src="https://user-images.githubusercontent.com/1464705/61564360-230ed100-aa2b-11e9-9671-b72f2af1d025.png">

---

At the top of /themes (this should link to Gutenberg to edit your homepage):

<img width="743" alt="Screen Shot 2019-07-19 at 1 29 13 PM" src="https://user-images.githubusercontent.com/1464705/61564402-3b7eeb80-aa2b-11e9-8caf-e4f51123ca5e.png">

---

In WP-Admin (this should be removed):

<img width="370" alt="Screen Shot 2019-07-19 at 1 30 09 PM" src="https://user-images.githubusercontent.com/1464705/61564423-46398080-aa2b-11e9-9006-5a6ce3bb52b9.png">

---

**Emails that we send that link to the Customizer:**

In our Welcome email (this should link to Gutenberg to edit your homepage):

<img width="521" alt="Screen Shot 2019-07-19 at 1 31 27 PM" src="https://user-images.githubusercontent.com/1464705/61564449-55203300-aa2b-11e9-9097-2788db6f4e78.png">

---

When I've purchased a plan or upgrade and get an email receipt (this should link to Gutenberg to edit your homepage):

<img width="646" alt="Screen Shot 2019-07-19 at 1 31 43 PM" src="https://user-images.githubusercontent.com/1464705/61564506-754ff200-aa2b-11e9-8564-7a7f2516f6f4.png">

---

When I see the "thank you" page in Calypso after purchasing a plan (this should link to Gutenberg to edit your homepage):

<img width="860" alt="Screen Shot 2019-07-19 at 1 34 04 PM" src="https://user-images.githubusercontent.com/1464705/61564557-8ef13980-aa2b-11e9-8e87-44a52846ac59.png">

---

When I visit the /plans page in Calypso and see links to what my plan offers (this should be removed):

<img width="579" alt="Screen Shot 2019-07-19 at 1 34 24 PM" src="https://user-images.githubusercontent.com/1464705/61564591-a03a4600-aa2b-11e9-8338-e8799318a628.png">

---

I will open another issue for dealing with plan features only available in the customizer.

**Comment by apeatling** [comment]
I'm sure I have missed other links to the customizer, so if you find other examples please add them to this issue.

**Comment by apeatling** [comment]
I've opened another issue for dealing with plan features in the customizer: https://github.com/Automattic/wp-calypso/issues/34792

**Comment by gwwar** [comment]
At a minimum we should update the theme activation link 👍

**Comment by mattwiebe** [comment]
@apeatling what about the action bar?

<img width="271" alt="image" src="https://user-images.githubusercontent.com/195089/62314416-8c91d500-b458-11e9-8ab1-8d77bf345cd3.png">

I would assume we'd want to target whichever page is being shown in that context… and also thank god the masterbar dropdown is gone :D

**Comment by mattwiebe** [comment]
The backend diff can be found at D31096-code

**Comment by apeatling** [comment]
Yes, this should be changed, good eye!

**Comment by mattwiebe** [comment]
Actually, since the pencil edit icon already takes us to Gutenberg, I'm just going to remove the Customize link for FSE sites.

**Comment by davemart-in** [comment]
Just noting that there will also be a link to the customizer on the Customer Home screen being release Aug 23rd.

**Comment by gwwar** [comment]
Thanks for the heads up @davemart-in, is that code already in? We can add another PR for it

**Comment by davemart-in** [comment]
Here's where the link is in our open PR: https://github.com/Automattic/wp-calypso/pull/35169/files#diff-6cd48476878b52c6f8387d0653fae7efR145 

cc @eliorivero and @pablinos  (just so you're aware of this)

**Comment by apeatling** [comment]
@mattwiebe Is this still in progress? Blocked on email testing? I'd like to get this closed out.

**Comment by mattwiebe** [comment]
Yup just email testing. I'm trying to sort that out today.

**Comment by mattwiebe** [comment]
Closing this one out after D31096-code shipped.
